feature/auth/src/main/java/com/mifos/feature/auth/login/LoginViewModel.kt

```diff
@@ -112,12 +117,11 @@ class LoginViewModel @Inject constructor(
         prefManager.saveToken("Basic " + user.base64EncodedAuthenticationKey)
         // Saving user
         prefManager.savePostAuthenticationResponse(user)
-
         if (prefManager.getPassCodeStatus()) {
             _loginUiState.value = LoginUiState.HomeActivityIntent
         } else {
             _loginUiState.value = LoginUiState.PassCodeActivityIntent
         }
+        _loginUiState.value = LoginUiState.HomeActivityIntent
```

Comment:
don't do this. Fix authNavgraph of Login Screen

Reply:
Bhai, i think this could be the right logic. lets connect in gmeet whenever you are free